eval.cpp

```diff
@@ -784,6 +793,11 @@ namespace Sass {
       Expression* ex = operator()(var);
       return evacuate_quotes(interpolation(ex));
 
+    } else if (Parent_Selector* var = dynamic_cast<Parent_Selector*>(s)) {
+
+      Expression* ex = operator()(var);
```

Comment:
In the visitors we usually call `->perform()` as a way to invoke the operator for that type. 
Perform essentially calls `operator()(foo)` with some sugar.

It also mean to you can perform other visitors on a object i.e. `foo->perform(&eval)`.

I did notice `operator()(foo)` is being used above, which is strange. Ideally we'd stick to using `->perform()` for consistency unless there was a reason not to.